connect.yaml

```diff
@@ -17,6 +17,9 @@ deployAs:
         - key: MAPPING_CONFIGURATION
           description: JSON serialised property mapping configuration
           required: false
+        - key: OTEL_EXPORTER_OTLP_ENDPOINT
+          description: OpenTelemetry OTLP exporter endpoint
+          required: true
```

Comment:
This (and the `OTEL_EXPORTER_OTLP_ENDPOINT_API_KEY`) should be optional.

Reply:
Ok - yeah, missed that in `localdev`, as `connect.yaml` is ingored. Think that would only manifest in the Connect installation workflow at the moment. Resolved,